flow-model-generator/modules/flow-model-generator-core/src/main/java/io/ballerina/flowmodelgenerator/core/AvailableNodesGenerator.java

```diff
@@ -361,7 +362,7 @@ private Optional<Category> getCategory(Symbol symbol, Predicate<ClassSymbol> con
                 String org = methodFunction.org();
                 String packageName = methodFunction.packageName();
                 String version = methodFunction.version();
-                boolean isHttpModule = org.equals(BALLERINA_ORG) && packageName.equals(HTTP_MODULE);
+                boolean isHttpModule = org.equals(BALLERINA) && packageName.equals(HTTP_MODULE);
```

Comment:
Just for more clarity I'd prefer to have name this constant as `BALLERINA_ORG`/ `ORG_BALLERINA`. But we can do it as a future improvement
```suggestion
                boolean isHttpModule = org.equals(BALLERINA) && packageName.equals(HTTP_MODULE);
```